script/ci-install-android-sdk.sh

```diff
@@ -30,7 +30,7 @@ else
 
     # Install Android SDK
     (echo "yes" | ${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager --licenses > /dev/null | grep -v = || true)
-    ( sleep 5; echo "y" ) | (${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager "build-tools;29.0.2" "platforms;android-29" > /dev/null | grep -v = || true)
+    ( sleep 5; echo "y" ) | (${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager "build-tools;30.0.3" "platforms;android-29" > /dev/null | grep -v = || true)
```

Comment:
maybe `android-30` ?

Reply:
There are no errors. This piece of code means what kind of an emulators we are going to use. We use an emulator of API 29 because it works better.